pkg/users/local.js

```diff
@@ -76,37 +77,45 @@ function AccountsPage() {
             return [];
     }, [accounts, details]);
 
-    const groupsExtraInfo = useMemo(() => sortGroups(
-        (groups || []).map(group => {
-            const userlistPrimary = accountsInfo.filter(account => account.gid === group.gid).map(account => account.name);
-            const userlist = group.userlist.filter(el => el !== "");
-            return ({ ...group, userlistPrimary, userlist, members: userlist.length + userlistPrimary.length, isAdmin: admins.includes(group.name) });
-        })
-    ), [groups, accountsInfo]);
+    const groupsExtraInfo = useMemo(() => {
+        console.debug("groupsExtraInfo is regenerated", groups, accountsInfo);
+        return sortGroups(
+            (groups || []).map(group => {
+                const userlistPrimary = accountsInfo.filter(account => account.gid === group.gid).map(account => account.name);
+                const userlist = group.userlist.filter(el => el !== "");
+                return ({ ...group, userlistPrimary, userlist, members: userlist.length + userlistPrimary.length, isAdmin: admins.includes(group.name) });
+            })
+        );
+    }, [groups, accountsInfo]);
 
     if (groupsExtraInfo.length == 0 || accountsInfo.length == 0) {
         return <EmptyStatePanel loading />;
     } else if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-133703-75368b81-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)